mu4/scenes/inspector/internal/compatibility/mu4inspectoradapter.cpp

```diff
@@ -0,0 +1,123 @@
+#include "mu4inspectoradapter.h"
+
+#include "log.h"
+
+using namespace mu::scene::inspector;
+using namespace mu::domain::notation;
+
+bool MU4InspectorAdapter::isNotationExisting() const
+{
+    return !context()->notations().empty();
```

Comment:
I think it is better to check for the current notation

Reply:
Both are valid, perhaps this one is a little more readable